x/erc20/keeper/proposals.go

```diff
@@ -88,6 +88,12 @@ func (k Keeper) RegisterERC20(
 		)
 	}
 
+	if len(metadata.DenomUnits) != 2 {
```

Comment:
added additional enforcement, so erc20 that wants to be registered must provide symbol name and decimals